releases/note-source/v14_0.rst

```diff
@@ -0,0 +1,373 @@
+.. _release-v14-0:
+
+Spring 2017 Release (v14_0)
+===========================
+
++-------------------------------------------+------------+
+| Source                                    | Identifier |
++===========================================+============+
+| Git tag                                   | 14.0       |
++-------------------------------------------+------------+
+| :doc:`EUPS distrib </install/newinstall>` | v14\_0     |
++-------------------------------------------+------------+
+
+- :ref:`release-v14-0-major-changes`
+- :ref:`release-v14-0-bug-fixes`
+- :ref:`release-v14-0-internal-improvements`
+
+.. - :ref:`release-v14-0-sui`
+.. - :ref:`Qserv and Data Access <release-v14-0-qserv-dax>`
+
+*See also:*
+
+- :doc:`Installation instructions <../install/index>`
+- :doc:`Known issues </known-issues>`
+- `Characterization Metric Report (DMTR-41) <https://ls.st/DMTR-41>`_
+
+These release notes highlight significant changes to the Science Pipelines codebase which are likely to be of wide interest.
+A complete list of issues addressed in this release is available :ref:`here <release-v14-0-tickets>`.
```

Comment:
Rather than the `here` like I'd suggest:

```
For a complete list of changes, see :ref:`release-v14-0-tickets`.
```

And let Sphinx populate the page title into the link. I think this'll have better usability than a "here" link.